vowpalwabbit/accumulate.cc

```diff
@@ -19,18 +19,21 @@ using namespace std;
 
```

Comment:
@alekh accumulate isn't factored right to support sparse models.  Can you think about how to do it right?